### DISCUSSION TITLE
Hosting on GCP

### DISCUSSION THREAD

**Discussion by obp3ter** [Cog-Creators/Red-DiscordBot#5229] [Q&A] [answered]
Hello, My question would be, if the free VM on GCP (e2-micro) is enough for the bot? Has anyone tried it?

**Comment by Jackenmen** [comment, marked as answer]
It depends on what cogs you plan to use. 

If you plan on using Audio or other demanding cogs, it will struggle. Oracle Cloud's Always Free tier is better than Google's and should be enough for running Audio on something like one Discord server but still don't expect miracles here.

Another thing to note here is that Google's free tier can have network egress costs if you use more than 1 GB per month and this is not an unlikely scenario if you use Audio cog.

See also: https://docs.discord.red/en/stable/host-list.html
You might also want to consider going to [our Discord server](https://discord.gg/red) though we do not provide hosting advice as part of #support (we only help with installing and using core Red). There are people who might be able to help you out in other channels though.

**Reply by obp3ter** [reply]
Thanks, I'll check it out. I was asking specifically Google because I'm deep in GCP and right now I'm not using the free VM for anything. But I'll have a look at Oracle as well. I didn't notice that part in the doc.
